snippets/quick-order-list-row.liquid

```diff
@@ -24,17 +24,18 @@
       <div class="variant-item__media">
         <div class="variant-item__image-container gradient global-media-settings{% unless image %} variant-item__image-container--no-img {% endunless %}">
           {% if image %}
-            {%- assign img_height = 45 | divided_by: image.aspect_ratio | ceil -%}
+            {%- assign img_height = 43 | divided_by: image.aspect_ratio | ceil -%}
             {{
               image
-              | image_url: width: 45
+              | image_url: width: 86
               | image_tag:
                 loading: 'lazy',
                 fetchpriority: 'low',
                 decoding: 'async',
                 class: 'variant-item__image',
-                width: 45,
+                width: 43,
```

Comment:
Value changed to `43` because, the actual width of the image on the page seems to be `43px`
<img width="639" alt="Screenshot 2023-07-14 at 2 48 43 PM" src="https://github.com/Shopify/dawn/assets/3710548/29f84623-9935-4c1e-83b1-76e9ea77903b">